ClientSyncComponent/ClientSyncComponent.Client/JsPuzzleChange.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace ClientSyncComponent.Client
+{
+    /// <summary>
+    /// Javascript entity for puzzle state matching puzzle.js
+    /// </summary>
+    public class JsPuzzleChange
+    {
+        public string locationKey { get; set; }
+        public string playerId { get; set; }
+        public string propertyKey { get; set; }
+        public string puzzleId { get; set; }
+        public string teamId { get; set; }
+        public string value { get; set; }
+        public string channel { get; set; }
```

Comment:
Maybe add a comment saying `channel` is largely NYI, since ClientSyncComponent doesn't take it into account when listening.

Reply:
Done